contrib/babelfishpg_tsql/src/forjson.c

```diff
@@ -101,7 +107,7 @@ tsql_query_to_json_internal(const char *query, int mode, bool include_null_value
 {
 	StringInfo	result;
 	uint64		i;
-
+	
```

Comment:
Please remove this.

Reply:
Although this was added keeping in mind the format of other functions in the same file, having a space after variable declarations, will remove the space in my next commit.